.github/workflows/push.yml

```diff
@@ -75,6 +75,11 @@ jobs:
         env:
           BUILDX_PLATFORM: ${{ matrix.platform }}
           BUILDX_BUILDER_NAME: ${{ steps.buildx-setup.outputs.name }}
+      - id: arm-time-limit
+        name: Set Netbox container start_period higher on ARM64
+        if: matrix.platform == 'linux/arm64'
+        run: |
+          sed -i -e 's/start_period: 120s/start_period: 180s/g' docker-compose.test.yml
       - id: docker-test
```

Comment:
A more robust way would be to use an Environment Variable in the dockercompose.test.yaml instead. It's default value would be the current value, and here you could just overwrite it. That'd be less fragile in case the content of dockercompose.test.yaml changes, e.g. if we change the default value.

Reply:
Yes, if this helps the tests I will build a better implementation. This was just the fastest way for a quick test.